figures/scripts/supp-S3-panels-BCD.R

```diff
@@ -0,0 +1,321 @@
+# Stephanie J. Spielman for CCDL, 2022
```

Comment:
Should we start adding more structure to the `figures/pdfs/supp` directory you think, e.g., adding a subdirectory for each supplemental figure?

Reply:
I think filing an issue about this would address this concern.

Reply:
Yes, I agree. I'll file an issue.